src/OpenTelemetry/Internal/DateTimeOffsetExtensions.net452.cs

```diff
@@ -57,6 +57,7 @@ public static DateTimeOffset FromUnixTimeMilliseconds(long milliseconds)
             {
                 throw new ArgumentOutOfRangeException(
                     nameof(milliseconds),
+                    milliseconds,
                     string.Format(CultureInfo.InvariantCulture, "milliseconds must be between {0} and {1}", MinMilliseconds, MaxMilliseconds));
```

Comment:
nit: just to be similar to others, can you change to interpolation?

Reply:
On it :)